opm/simulators/wells/WellInterface.hpp

```diff
@@ -165,6 +165,15 @@ namespace Opm
                                     Opm::DeferredLogger& deferred_logger
                                     ) = 0;
 
+        virtual void assembleWellEqWithoutIteration(const Simulator& ebosSimulator,
```

Comment:
As mentioned above, we do not see we need to introduce this function in the base class from this PR. Probably it happens because of other development.  Anyway, we should introduce the change based on the purpose of the PR itself.

Reply:
Now I see you use this function in `solveWellEqUntilConverged`. Depending the later decision on how to do `solveWellEqUntilConverged`, we can decide whether to keep this function in the base class. 

In either way, this function does not need to be `public`.

Reply:
I have made it private